src/Formplot/Formplot.csproj

```diff
@@ -9,7 +9,7 @@
     <Copyright>Copyright © $([System.DateTime]::UtcNow.Year) Carl Zeiss Industrielle Messtechnik GmbH</Copyright>
     <Company>Carl Zeiss Industrielle Messtechnik GmbH</Company>
     <Authors>$(Company)</Authors>
-    <PackageVersion>3.0.7</PackageVersion>
+    <PackageVersion>3.1.0</PackageVersion>
```

Comment:
Release pipeline will change that. Please remove it.

Reply:
Done